test/acceptance/integration/IOTCORE-124 ServicePath for registrations (NGSI9) (not recursive)/iotcore-124.feature

```diff
@@ -0,0 +1,450 @@
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And build the standard entity update payload with the previous data
+    When a standard context entity update is asked with the before information
+    Then retrieve information from the mock
+    And the path in the last mock petition contains "service1"
+    And there is "1" petitions requested to the mock
+    And clean the mongo database of the service "iotcore_124"
+
+  @iotcore-124
+  Scenario: Append an entity in a context broker, having context providers with the same entity in the same subservice and other in other subservice
+    Given a started mock
+    # First registration
+    And a new NGSI version "9" petition with the service "iotcore_124" and the subservice "/subservice1"
+    And the following entities to consult
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And the following attributes to consult
+      | attribute_name | attribute_type |
+      | att1           | att_type_1     |
+    And a context registrations with the before entities and attributes and the following providing applications
+      | providing_application      |
+      | /context_provider/service1 |
+    And build the standard context registration payload with the previous data and duration "P1M"
+    And a standard context registration is asked with the before information
+    # Second Registration
+    And a new NGSI version "9" petition with the service "iotcore_124" and the subservice "/subservice2"
+    And the following entities to consult
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And the following attributes to consult
+      | attribute_name | attribute_type |
+      | att1           | att_type_1     |
+    And a context registrations with the before entities and attributes and the following providing applications
+      | providing_application      |
+      | /context_provider/service2 |
+    And build the standard context registration payload with the previous data and duration "P1M"
+    And a standard context registration is asked with the before information
+    # Append operation
+    And a new NGSI version "10" petition with the service "iotcore_124" and the subservice "/subservice1"
+    And the following attributes to create
+      | attribute_name | attribute_type | attribute_value |
+      | att1           | att_type_1     | 25              |
+    And a context elements with the before attrs and the following entities
+      | entity_id | entity_type |
+      | Room1     | Room        |
+    And build the standard entity creation payload with the previous data
+    When a standard context entity creation is asked with the before information
+    Then retrieve information from the mock
+    And there is "0" petitions requested to the mock
+    And clean the mongo database of the service "iotcore_124"
+
+# Falla
+  @issue_755 @iotcore-124
```

Comment:
755 its the id of the bug that correspond with this tests, the iotcore is deleted and is replaced by the reference of the issue in github